applications/conceptual-model-editor/src/diagram/node/entity-node.tsx

```diff
@@ -89,31 +90,95 @@ function SelectedHighlight() {
 
 function EntityNodeToolbar(props: NodeProps<Node<ApiNode>>) {
   const context = useContext(DiagramContext);
+
+  // If canvas toolbar is opened then don't show toolbar
+  if((context?.openedCanvasToolbar ?? null) !== null) {
+    return <></>;
+  }
+
+  if(context?.shouldShowSelectionToolbar()) {
```

Comment:
Why don't you check that context is defined before? Now you need to deal with this throughout the function.
It also means that the second one is used then shouldShow... is false OR context is undefined, which I belive is not what you want to do here.